.envrc

```
@@ -0,0 +1,9 @@
+if ! has nix_direnv_version || ! nix_direnv_version 2.3.0; then
+    source_url "https://raw.githubusercontent.com/nix-community/nix-direnv/2.3.0/direnvrc" "sha256-Dmd+j63L84wuzgyjITIfSxSD57Tx7v51DMxVZOsiUD8="
+fi
+
+use flake . --impure
+
```

I want to suggest adding this now that we're at it. Many have asked about this on Slack the last years.

```suggestion
if [ -f .env ]
then
  set -o allexport
  source .env
  set +o allexport
fi
```

That would allow adding environments for the dev environments that are not checked into source control. So also `.env` would need to be added to `.gitignore`

If the `.env` file does not exist, nothing needs to be done, and one can still use `export` as usual if one prefers this

---

Yes, this seems useful to avoid future pain 👍